### DISCUSSION TITLE
[Projects beta]: add Discussions to board

### DISCUSSION THREAD

**Discussion by bennypowers** [community/community#16038] [Projects and Issues] [answered]
Sometimes we're not ready to open issues (i.e. specific action items), rather we want to discuss a feature or change before committing to implementing it. Discussions is good for that, so we'd like to be able to add a discussion to the project board.

**Comment by bennypowers** [comment, marked as answer]
This is on the roadmap: https://github.com/github/roadmap/issues/275

**Reply by michael-hawker** [reply]
Any update on this? How do we know from the roadmap when a feature will be worked on?
